docker/Dockerfile

```diff
@@ -1,10 +1,6 @@
 FROM ruby:3.2.2
 
-WORKDIR /srv/app/datagovuk_tech_docs
-
-RUN apt-get update
-RUN apt-get install -y nodejs
```

Comment:
Are you sure nodejs isn't needed? I tried it before without it and it seemed to crash

Reply:
Yep you're right the node environment is needed because of the coffee-script gem. I tried to a multi-stage build but that turned out to be far more complicated.